contrib/for-tests/mount-tester/mt.go

```diff
@@ -143,3 +151,30 @@ func readWriteNewFile(path string) error {
 
 	return readFileContent(path)
 }
+
+func readFileContentInLoop(path string) error {
```

Comment:
This function only checks that >0 bytes can be read from the file, and is not configurable with respect to how many bytes are expected, how long to retry for etc.

Would it not make sense to have it take bytesExpected and retryDuration as input parameters to make it more generally useful?  And then perhaps move it to one of the utils packages (e.g. testing/utils)?